### PR TITLE
[Docs][SIEM] 7.7 detection rule updates

### PR DESCRIPTION
**Pull request by benskelker**
Updates the docs with added detections functionality.

[Preview](http://stack-docs_974.docs-preview.app.elstc.co/guide/en/siem/guide/master/detection-engine-overview.html)

### CODE REVIEW

File: docs/en/siem/machine-learning.asciidoc [hunk 1]
```diff
@@ -20,9 +20,9 @@ image::ml-ui.png[]
 [float]
 [[manage-jobs]]
 === Manage machine learning jobs
-For users with the `ml_admin` role, the `Anomaly Detection` interface within
-the main navigation header can be used for for viewing, starting, and stopping
-SIEM machine learning jobs.
+For users with the `machine_learning_admin` role, the `Anomaly Detection`
```
Comment (rylnd) [inline docs/en/siem/machine-learning.asciidoc:23]: I'm sure you're aware, but with #62396 this is changing to `ML job settings` and will only be visible on the Detections page.

File: docs/en/siem/machine-learning.asciidoc [hunk 2]
```diff
@@ -20,9 +20,9 @@ image::ml-ui.png[]
...
+interface within the main navigation header can be used for for viewing,
+starting, and stopping SIEM machine learning jobs.
 
 TIP: To add a custom job to the `Anomaly Detection` interface, add a `SIEM` tag
```
Comment (rylnd) [inline docs/en/siem/machine-learning.asciidoc:27]: This isn't new, but I think that the same feedback as [this comment](https://github.com/elastic/siem-team/issues/519#issuecomment-608583504) applies, where the verb "tag" is potentially misleading.

Comment (benskelker) [reply docs/en/siem/machine-learning.asciidoc:27]: Thanks @rylnd - I'll update it

File: docs/en/siem/rules-ui-create.asciidoc [hunk 3]
```diff
@@ -70,25 +71,41 @@ ones.
...
-image::create-new-rule.png[]
-. Define which {es} indices the rule analyzes for signals.
-. Use the filter and query fields to create the criteria used for detecting 
+image::images/create-new-rule.png[]
+. Do *one* of the following:
+* To create a rule based on a {ml} anomaly threshold, select _Machine Learning_
+and then:
+.. The required {ml} job.
+.. The anomaly score threshold above which signals are created.
+* To create a rule base on a KQL or Lucene query, select _Custom query_ and
```
Comment (dhurley14) [inline docs/en/siem/rules-ui-create.asciidoc:94]: I think you want `base` to be `based` here.

```suggestion
* To create a rule based on a KQL or Lucene query, select _Custom query_ and
```

Comment (benskelker) [reply docs/en/siem/rules-ui-create.asciidoc:94]: sure do - thanks @dhurley14

File: docs/en/siem/machine-learning.asciidoc [hunk 4]
```diff
@@ -20,12 +20,12 @@ image::ml-ui.png[]
 [float]
 [[manage-jobs]]
 === Manage machine learning jobs
-For users with the `ml_admin` role, the `Anomaly Detection` interface within
-the main navigation header can be used for for viewing, starting, and stopping
-SIEM machine learning jobs.
+For users with the `machine_learning_admin` role, the `ML job settings`
```
Comment (spong) [inline docs/en/siem/machine-learning.asciidoc:23]: Thanks for the distinction here as we've (myself at least 😅) been conflating the users/roles -- `ml_admin` is the user, with the roles being `machine_learning_admin`, `machine_learning_user`, etc.

File: docs/en/siem/rules-ui-create.asciidoc [hunk 5]
```diff
@@ -99,11 +116,17 @@ image::rule-query-example.png[]
 TIP: This example is based on the
 <<volume-shadow-copy-deletion-via-vssadmin, Volume Shadow Copy Deletion via VssAdmin>> prebuilt rule.
 
+. Select the timeline template used when you  send a signal created by the rule
```
Comment (spong) [inline docs/en/siem/rules-ui-create.asciidoc:119]: Docs render fine of course, but extra space here:

```suggestion
. Select the timeline template used when you send a signal created by the rule
```

**Pull request merged**